packages/workspace/src/schematics/preset/preset.ts

```diff
@@ -80,12 +97,20 @@ function createPreset(options: Schema): Rule {
       setDefaultCollection('@nrwl/web'),
     ]);
   } else if (options.preset === 'angular-nest') {
+    // Remove Codelyzer from package.json if chosen linter is ESLint
+    let packageUpdates = noop();
+    if (options.linter === 'eslint') {
+      packageUpdates = removeCodelyzerFromDevDependencies();
```

Comment:
Angular init doesn't have a linter option, that's the exact reason why I applied it where I did :)

Was trying to avoid any major refactoring.

Are you saying you would prefer me to add a linter option to Angular init as part of this PR?

Reply:
Yeah, I think it should take a `linter` option. It already takes a `unitTestRunner` and `e2eTestRunner` so it makes sense to include `linter` there. I don't imagine it would be a large refactor so can you give it a try?